components/tableNodes/NodeTable.component.tsx

```diff
@@ -94,11 +94,13 @@ export function NodeTable({ show }: NodeTableProps): JSX.Element {
                   {}
                 </TableCell>
                 <TableCell component="th" scope="row">
-                  {channel.amount_msat + " msats"}
+                  {channel.amount_msat + "s"}
```

Comment:
what is this `s`?

Reply:
because `channel.amount_msat` returns `<any number> msat` so added an "s" at the end to make "msats"

Reply:
when it is an int (if we disable the deprecated API) the `msat` is needed, so always when you make a decision in the UIX you need to take the decision to not crash in some things that change in the API and you are crashing.

We need to have a function that returns a string, and checks the `amount_msat` has a `msat` at the end or not, if not we will append `msat` otherwise we append nothing.

the `s` is not needed

Reply:
Done the changes